keps/sig-apps/19-Graduate-CronJob-to-Stable/kep.yaml

```diff
@@ -26,12 +26,12 @@ replaces:
   - "https://github.com/kubernetes/community/blob/master/contributors/design-proposals/apps/cronjob.md"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: stable
```

Comment:
You need a prod-readiness file, as this one:
https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-apps/85.yaml

I will make a PRR review tomorrow (though I remember reviewing that in the past, so don't expect any significant changes needed).

Reply:
Yeah, I've noticed after initial push and updated that with the right one now.